appcelerator/titanium_mobile
# fix: convertPointToView() to apply ScrollView contentOffset

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28398

**Summary:**
If calling `childView.convertPointToView(point, scrollView)`, when the destination is a `ScrollView`, then its `contentOffset` should be applied to the returned value. This would make it easier to scroll to a child view within a scroll view.

**Test:**
_(Run this test on both Android and iOS.)_
1. Build and run `ScrollViewOffsetVerticalTest.js` attached to [TIMOB-28398](https://jira.appcelerator.org/browse/TIMOB-28398).
2. Tap on the "Navigate" button in the bottom right corner.
3. Select "Blue View" in the dialog.
4. Verify the top of the blue view is scrolled to the top.
5. Tap on the "Navigate" button and select "Blue View" again.
6. Verify the top of the blue view is still displayed. It should not scroll at all. _(This was a bug.)_
7. Tap on the "Navigate" button and select "Red View".
8. Verify you are scrolled to the top of the red view.
